src/app/utils/global-error-handler.ts

```diff
@@ -11,7 +11,7 @@ export class GlobalErrorHandler implements ErrorHandler {
         }
         win.document.open();
         win.document.write(
-            `<head><title>Error Report</title></head><body>An Error occured. Sorry for the inconvenience. Error Message: <br/> ${ErrorSerializer.serailize(
+            `<head><title>Error Report</title></head><body>An Error occured. Sorry for the inconvenience. <br/>Error Message: <br/> ${ErrorSerializer.serailize(
```

Comment:
Currently, the error report does not respect newline characters in the error message. Add `white-space: pre-line` to the style of the error message container could resolve this issue.

Reply:
Also, add `overflow-wrap: anywhere` to the style of the error messaeg container could make the error message better shown in the popup window and simplify copy & pasting the error message.